From: Ahmad Fatoum <a.fatoum@pengutronix.de>
<<<<<<<< HEAD:meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1701-Release-6.9-customers-skov-imx6-20240412-2.patch
Date: Fri, 12 Apr 2024 22:32:11 +0200
Subject: [PATCH] Release 6.9/customers/skov/imx6/20240412-2
|||||||| parent of 2b981ad17213 (update):meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1501-Release-6.7-customers-skov-imx6-20240411-2.patch
Date: Thu, 11 Apr 2024 14:37:30 +0200
Subject: [PATCH] Release 6.7/customers/skov/imx6/20240411-2
========
Date: Fri, 12 Apr 2024 14:55:07 +0200
Subject: [PATCH] Release 6.9/customers/skov/imx6/20240412-1
>>>>>>>> 2b981ad17213 (update):meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1701-Release-6.9-customers-skov-imx6-20240412-1.patch

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
<<<<<<<< HEAD:meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1701-Release-6.9-customers-skov-imx6-20240412-2.patch
index e1bf12891cb0..51cd23e56bf2 100644
|||||||| parent of 2b981ad17213 (update):meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1501-Release-6.7-customers-skov-imx6-20240411-2.patch
index c6f549f6a4ae..c438f7bf6f0e 100644
========
index e1bf12891cb0..e834108bc69e 100644
>>>>>>>> 2b981ad17213 (update):meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1701-Release-6.9-customers-skov-imx6-20240412-1.patch
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 9
 SUBLEVEL = 0
<<<<<<<< HEAD:meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1701-Release-6.9-customers-skov-imx6-20240412-2.patch
-EXTRAVERSION = -rc3
+EXTRAVERSION = -rc3-20240412-2
|||||||| parent of 2b981ad17213 (update):meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1501-Release-6.7-customers-skov-imx6-20240411-2.patch
-EXTRAVERSION =
+EXTRAVERSION =-20240411-2
========
-EXTRAVERSION = -rc3
+EXTRAVERSION = -rc3-20240412-1
>>>>>>>> 2b981ad17213 (update):meta-skov-bsp/recipes-kernel/linux/linux-skov/patches/1701-Release-6.9-customers-skov-imx6-20240412-1.patch
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
